java/src/jmri/jmrix/dccpp/swing/DCCppSwingBundle.properties

```diff
@@ -9,6 +9,12 @@ MenuItemConfigBaseStation = Configure Base Station
 MenuItemDCCppOverTCPServer = Load DCC++ over TCP Server
 DCCppMonFrameTitle = DCC++ Traffic Monitor
 
+MenuSend     = Send
+RequestDefs  = Request Defs
```

Comment:
What does "Def" stands for? Maybe it's obvious for a DCCpp user.
I'm not a native English speaker so maybe that's causing my confusion.

Reply:
Definition.

Reply:
Thanks. It might be good to change the label to `Request definitions` to make it clearer, but I'm happy to merge the PR as is.